src/pages/DeckDetail/CardItems.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import Button from '../../components/UI/Button/Button'
+import classes from './CardItems.module.css'
+import { useHistory } from 'react-router-dom'
+import { useSelector } from 'react-redux'
+import { useState } from 'react'
+import axios from '../../axios/axiosInstanceFunction'
+import TrashIcon from '../../assets/images/trash.svg'
+import EditIcon from '../../assets/images/edit.svg'
+
+const CardItems = ({ deckId }) => {
+    const history = useHistory()
+    const { cardDetails: cardList } = useSelector((store) => store.deck)
+    const [selectedIds, setSelectedIds] = useState([])
+
+    const onClickEditHandler = (id) => {
+        history.push(`/card/${id}`)
+    }
+
+    const onClickDeleteHandler = async (id) => {
+        console.log('deleted', id)
+        try {
+            const response = await axios('http://168.138.215.26:9002/').delete(
+                `/card/${id}`
+            )
+            console.log(response)
+        } catch (err) {
+            console.log(err)
+        }
+    }
+
+    const onSelectCheckbox = (e, id) => {
+        if (e.target.checked) {
+            setSelectedIds([...selectedIds, id])
+        } else {
+            setSelectedIds(
+                selectedIds.filter((current_id) => current_id !== id)
+            )
+        }
+        console.log(selectedIds)
+    }
+
+    const onClickDeleteSelected = () => {
```

Comment:
I think maybe this was for testing purposes but I think telling the user just that the card was deleted or the name of the card may be better. Additionally, while the edit side button works the trash can button doesn't. The deleted card still appears even after refresh of the page as well.